dev-tools/composer.json

```diff
@@ -7,7 +7,8 @@
         "localheinz/composer-normalize": "^1.1",
         "maglnet/composer-require-checker": "^2.0",
         "mi-schi/phpmd-extension": "^4.3",
-        "phpmd/phpmd": "^2.6"
+        "phpmd/phpmd": "^2.6",
+        "phpstan/phpstan": "^0.11"
```

Comment:
we might want;
```
phpstan/phpstan-deprecation-rules
phpstan/phpstan-strict-rules
phpstan/phpstan-phpunit
phpstan/phpstan-php-parser
```

as well?

Reply:
Indeed, but let's add them later, just as we'll rise the analyse level later.